phpstan/config/default.neon

```diff
@@ -0,0 +1,24 @@
+parameters:
+	level: 6
+	bootstrapFiles:
+		- ../constants.php
+	excludePaths:
+		- ../../*/vendor/*
+		- ../../*/dist/*
+		- ../../*/node_modules/*
+		- ../../*/tests/*
+		- ./*
+	scanDirectories:
+		- ../../plugins
+	scanFiles:
+		- ../../vendor/php-stubs/wordpress-stubs/wordpress-stubs.php
+		- ../../vendor/php-stubs/wp-cli-stubs/wp-cli-stubs.php
+		- ../../vendor/php-stubs/wp-cli-stubs/wp-cli-commands-stubs.php
+		- ../../vendor/php-stubs/wp-cli-stubs/wp-cli-i18n-stubs.php
+	stubFiles:
+		- ../stubs/wp-query.stub
+	ignoreErrors:
```

Comment:
@darylldoyle Having `ignoreErrors` in the default config throws errors if the project does not have these types of errors. I like the idea of having a default config especially if this is extracted into its own Composer package much like `phpcs-composer`. Moving these to the project config is the easiest fix but then we will lose that company-wide default config.

Reply:
These are common errors in WordPress so I think we should keep them. It looks like there's the ability to turn off the reporting of unmatched error ignores, see [here](https://phpstan.org/user-guide/ignoring-errors#reporting-unused-ignores).

Reply:
These will no longer error